jinad/models/pod.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional
+from pydantic import BaseModel, Field, validator
+
+PRE_DEFINED_USES = ['_clear', '_pass', '_route', '_merge', '_logforward']
+
+    
+class PodBase(BaseModel):
```

Comment:
Idea:

When this grows a little, keep some `json` description of the `jina` classes parameters for these `model` classes to read from, so that this gets more mantainable and have the core be responsible for keeping it up-to-date. Otherwise these models will struggle to keep up with the core changes